static/app/views/settings/project/filtersAndSampling/modals/form.tsx

```diff
@@ -147,7 +147,9 @@ class Form<P extends Props = Props, S extends State = State> extends React.Compo
     // DynamicSamplingConditionLogicalInnerGlob
     if (
       condition.category === DynamicSamplingInnerName.EVENT_RELEASE ||
-      condition.category === DynamicSamplingInnerName.TRACE_RELEASE
+      condition.category === DynamicSamplingInnerName.TRACE_RELEASE ||
+      condition.category === DynamicSamplingInnerName.EVENT_TRACE ||
+      condition.category === DynamicSamplingInnerName.TRACE_TRANSACTION
```

Comment:
at some point might consider doing `[DynamicSamplingInnerName.EVENT_RELEASE, TRACE_RELEASE].includes(condition.category)`

Reply:
sure.. that is a good idea! I'll update the code..thanks @scttcper

Reply:
hm if I do this I will have to cast the return because the compiler is not happy,  so I think I prefer to leave as it is